www/artists/dhall/photographs.html

```diff
@@ -0,0 +1,3 @@
+<li> <a href= "index.html"> Home</a> </li>
```

Comment:
...This page lacks all the vital elements of an HTML file. No head, no body, _no HTML tag_...